conda/base/context.py

```diff
@@ -613,12 +613,18 @@ def conda_exe_vars_dict(self):
         '''
 
         if context.dev:
-            return OrderedDict([('CONDA_EXE', sys.executable),
-                                ('PYTHONPATH', os.path.dirname(CONDA_PACKAGE_ROOT) + '{}{}'.format(
-                                    os.pathsep, os.environ.get('PYTHONPATH', ''))),
-                                ('_CE_M', '-m'),
-                                ('_CE_CONDA', 'conda'),
-                                ('CONDA_PYTHON_EXE', sys.executable)])
+            return OrderedDict(
+                [
+                    ("CONDA_EXE", sys.executable),
+                    (
+                        "PYTHONPATH",
+                        os.pathsep.join((CONDA_SITE_PACKAGES, os.environ.get("PYTHONPATH", ""))),
```

Comment:
This is a bit weird since there is a very similar main API of `os.path.join(path, components, etc)` that has an uncanny similarity to this way of writing it. Could we use that instead or add a comment why we're not using it?

Reply:
So this is using `os.pathsep` (e.g. `;` and `:`) not `os.sep` (e.g. '/' and '\') so I don't think there's any way to use `os.path.join` here?

Reply:
But yes a comment highlighting this special case here would be good.

Reply:
Oh, d'oh, you're right, I misread the line! Thanks for the context :)